src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"beige", "magenta", "blue", "yellow", "pink", "purple"};
+    private static final Random RANDOM = new Random();
```

Comment:
reference-type variables should be fields, not constants
```suggestion
    private final Random random = new Random();
```